pkg/apis/core/v4beta1/requirements_test.go

```diff
@@ -35,6 +35,7 @@ func TestRequirementsConfigMarshalExistingFile(t *testing.T) {
 	expectedClusterName := "my-cluster"
 	expectedSecretStorage := v4beta1.SecretStorageTypeVault
 	expectedDomain := "cheese.co.uk"
+	expectedExtraDomains := []string{"wine.com", "grapes.org.uk"}
```

Comment:
If you do change ExtraDomains to be a pointer then you'd need to change this to be
```
expectedExtraDomains := &[]string{"wine.com", "grapes.org.uk"}
```